aries_cloudagent/protocols/out_of_band/v1_0/messages/problem_report.py

```diff
@@ -64,6 +66,13 @@ class Meta:
         model_class = OOBProblemReport
         unknown = EXCLUDE
 
+    _type = fields.Str(
```

Comment:
Should this be part of the base AgentMessageSchema?

Reply:
`_type` is already part of AgentMessageSchema but that has `dump_only=True` which gets removed in `BaseModel  -> skip_dump_only` when deserializing. So I am overwriting `_type` only for these OOB messages. I updated `BaseModel -> make_model` to adjust for the above. The reason for these updates is to achieve backward compatibility, earlier if we received `1.0` message type and version found in `message_types` was `1.1` then the model `_type` was always `1.1`.

Reply:
Ah okay, that makes sense.